lib/import.php

```diff
@@ -86,7 +87,20 @@ public function master() {
 	 *
 	 * @return string|WP_Error
 	 */
-	protected function commit( $commit ) {
+	protected function commit( $commit , $commits_array) {
+		$head_commit = array();
+		$mod_files = array();
+		$update_all = true;
+		foreach ( $commits_array as $commit_obj ) {
+			if ( $commit_obj != null ) {
+				$next_commit = json_decode(json_encode($commit_obj), true);
```

Comment:
We can cast the object to an array with `$next_commit = (array) $commit_obj;` instead of the whole JSON thing.